test/CMakeLists.txt

```diff
@@ -169,6 +169,9 @@ if (USE_YT_BASED_TESTS)
   setup_test_serial_python(vlct_passive_advect_sound vlct "input/vlct/run_passive_advect_sound_test.py")
   setup_test_parallel_python(vlct_dual_energy_shock_tube vlct "input/vlct/run_dual_energy_shock_tube_test.py")
 
+  # Gravity (with VLCT)
+  #setup_test_serial_python(gravity_vlct_stable_Jeans_wave gravity "input/Gravity/run_stable_jeans_wave_test.py")
```

Comment:
Any reason not to include this test as part of the standard set?

Reply:
This is on my ToDo list to fix. I need to update the expected answers